docs/book/releases/v2.1.0.md

```diff
@@ -51,6 +51,11 @@
 7. When a Pod is rescheduled to a new node, there may be some lock contention which causes a delay in the volume getting detached from the old node and attached to the new node.
    - Impact: Rescheduled Pods remain in `Pending` state for an varying amount of time.
    - Workaround: Upgrade CSI driver to `v2.1.1`.
+8. When pod using a PVC is rescheduled to other node when metadatasyncer is down, fullsync might fail with error `Duplicated entity for each entity type in one cluster is found`.
+   - Impact: CNS may hold stale volume metadata.
+   - Workaround:
+      - Unregister this volume with CNS and then register this volume with CNS.
```

Comment:
I think let's just suggest upgrading CSI driver, if the customer is having a lot of Pods in this state, un-register, and register may create many other issues.

Reply:
Fixed.